### ISSUE TITLE
Make desktop notifications come from Biofabforum

### ISSUE THREAD

**Issue by tanius** [edgeryders/discourse-theme-biofabforum#4]
Right now, desktop notification overlays will show the Edgeryders logo when a user is logged on biofabforum.org, whether or not the content is placed in the Biofabforum category or a "normal" Edgeryders category. (Users who participate in discussion in other categories on egderyders.eu might get desktop notifications about such content while logged in on biofabforum.org.)

Instead, the desktop notification should show the Biofabforum logo in case of content in the Biofabforum category, or in case of all desktop notifications during a visit on biofabforum.org, whichever is simpler to implement.

**Comment by tanius** [comment]
As an initial solution, we could also hide the desktop notification setting on biofabforum.org. It's not an essential feature, so we can easily add the proper implementation later …

**Comment by tanius** [comment]
This issue was solved by Apache `mod_rewrite` redirections from `/images/logo-small.png` to `/images/logo-small.biofabforum.org.png`. We had set up these to fix the random logo changes on the site, and it also solved the issue with desktop notifications as a side effect.

Now, desktop notifications will always bear the correct logo of the browser tab they are coming from. Discourse keeps up an active connection to exactly one tab in a browser to send updates about new posts (which then appear as blue bubble notifications and desktop notifications). So if you use both the edgeryders.eu and biofabforum.org tabs in parallel, it is not determined which one will keep the active connection, that is, which logo you will see on the desktop notifications. But that  is ok, as in such a case you will know what you're doing.
